invesalius/navigation/navigation.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, vis_queues, vis_components, event, sle):
 
         threading.Thread.__init__(self, name='UpdateScene')
         self.serial_port_enabled, self.view_tracts, self.peel_loaded = vis_components
-        self.coord_queue, self.serial_port_queue, self.tracts_queue, self.icp_queue = vis_queues
+        self.coord_queue, self.serial_port_queue, self.tracts_queue, self.icp_queue, self.robottarget_queue = vis_queues
```

Comment:
rename: `robot_target_queue`
do we need the word `target` here? Can it be just `robot_queue`?